src/integrationTest/java/integration/tests/SystemEngineDatabaseMetaDataTest.java

```diff
@@ -29,6 +25,8 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class SystemEngineDatabaseMetaDataTest extends IntegrationTest {
+    private static final long ID = ProcessHandle.current().pid() + System.currentTimeMillis();
+    private static final String DATABASE_NAME = "jdbc_system_engine_integration_test_" + ID;
```

Comment:
Current `database` name is already unique when tests run from the integration tests so we can avoid doing this and just reuse the name + some suffix.